api/onnx_web/diffusers/run.py

```diff
@@ -294,6 +357,12 @@ def run_inpaint_pipeline(
 
     _pairs, loras, inversions, _rest = parse_prompt(params)
     for image, output in zip(images, outputs):
+        if full_res_inpaint:
+            if is_debug():
+                save_image(server, "adjusted-output.png", image)
+            mini_image = ImageOps.contain(image, (adj_mask_size, adj_mask_size))
+            image = original_source
```

Comment:
Is `original_source` still defined correctly when `full_res_inpaint is True` but `border.isZero() is False` on https://github.com/ssube/onnx-web/pull/397/files#diff-4a1fc49eb8b8b908d1b14572baa04035c425791367c2a54f3f728ce35d1dc33aR255?

Reply:
It sure does not. Fixed.